lib/irc/BridgedClient.js

```diff
@@ -333,6 +333,51 @@ BridgedClient.prototype.whois = function(nick) {
     });
 };
 
+
+/**
+ * Get the operators of a channel
+ * @param {string} channel : The channel to call /names on
+ */
+BridgedClient.prototype.getOperators = function(channel) {
+    let modify = function (nicksInfo) {
+        let nicks = nicksInfo.nicks;
+
+        nicksInfo.operatorNicks = nicks.filter(
+            (index) => {
```

Comment:
`filter()` calls the function for each element in the array, not the index, so this variable name is misleading. Please rename it to something less misleading e.g. `nick`.